packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -20,7 +26,9 @@ spec:
           - name: ISTIO_NAMESPACE
             value: {{ template "istio.namespace" . }}
           - name: FORCE_INSTALL
-            value: {{ .Values.forceInstall | default "false" | quote }}
+            value: {{ .Values.installer.forceInstall | quote }}
+          - name: RELEASE_MIRROR_ENABLED
+            value: {{ .Values.installer.releaseMirror.enabled | quote }}
```

Comment:
should a default be introduced here too?

Reply:
default cannot be introduced here because the value is used for an if check else where and I don't want inconsistencies in how that evaluates. Its also set by default in the values.yaml so unless the user removes the value all together then it will be set.

Reply:
sounds good to me. I don't think dropping it is that much of a concern.